General improvements fulfillment guide (#267)
vtexdocs/dev-portal-content: docs/guides/Getting-Started/fulfillment.md

2. [Loading dock](https://help.vtex.com/en/tutorial/managing-loading-docks--7K3FultD8I2cuuA6iyGEiW)
3. [Warehouse](https://help.vtex.com/en/tutorial/warehouse--6oIxvsVDTtGpO7y6zwhGpb)

**In VTEX Admin:** Shipping > Shipping Strategy > Shipping Policies/Loading Docks/Warehouses

### 1. Shipping policy

> [Shipping policy](https://help.vtex.com/en/tutorial/shipping-policy--tutorials_140) defines the shipping conditions offered to the shopper at checkout, including costs and deadlines. This shipping policy set of rules includes carriers working hours, shipping capacity, and more.

Reviewer: ```suggestion
A [Shipping policy](https://help.vtex.com/en/tutorial/shipping-policy--tutorials_140) defines the shipping conditions offered to the shopper at checkout, including costs and deadlines. This set of rules includes carriers working hours, shipping capacity, and more.
```

* [POST - Create shipping policy](https://developers.vtex.com/docs/api-reference/logistics-api#post-/api/logistics/pvt/shipping-policies)

### 2. Loading dock

> [Loading dock](https://help.vtex.com/en/tutorial/managing-loading-docks--7K3FultD8I2cuuA6iyGEiW) connects the warehouse to the shipping policy. The loading dock corresponds to a physical location from where products will be shipped. By default, when you create a new loading dock, it is generated as activated.

Reviewer: ```suggestion
[Loading docks](https://help.vtex.com/en/tutorial/managing-loading-docks--7K3FultD8I2cuuA6iyGEiW) connect the warehouse to the shipping policy. The loading dock corresponds to a physical location from where products will be shipped. By default, when you create a new loading dock, it is generated as activated.
```

* [POST - Create/update dock](https://developers.vtex.com/docs/api-reference/logistics-api#post-/api/logistics/pvt/configuration/docks)
* [POST - Activate dock](https://developers.vtex.com/docs/api-reference/logistics-api#post-/api/logistics/pvt/configuration/docks/-dockId-/activation)

### 3. Warehouse

> [Warehouse](https://help.vtex.com/en/tutorial/warehouse--6oIxvsVDTtGpO7y6zwhGpb) is the physical location where you store your products. The items sold go from the warehouse to the loading dock for the carriers to deliver them. By default, when you create a new warehouse, it is generated as activated.

Reviewer: ```suggestion
A [Warehouse](https://help.vtex.com/en/tutorial/warehouse--6oIxvsVDTtGpO7y6zwhGpb) is the physical location where you store your products. The items sold go from the warehouse to the loading dock for the carriers to deliver them. By default, when you create a new warehouse, it is generated as activated.
```

* [POST - Create/update warehouse](https://developers.vtex.com/docs/api-reference/logistics-api#post-/api/logistics/pvt/configuration/warehouses)
* [POST - Activate warehouse](https://developers.vtex.com/docs/api-reference/logistics-api#post-/api/logistics/pvt/configuration/warehouses/-warehouseId-/activation)

> ## Managing Shipping Strategy

Reviewer: ```suggestion
## Managing shipping strategy
```

(para ser condizente com a grafia que aparece antes no guia)

> Manage your [Shipping strategy](https://help.vtex.com/en/tutorial/shipping-strategy--58vLBDbjYVQzJ6rRc5QNz3) by editing your shipping policies, loading docks, and warehouses.

Reviewer: Vale padronizar a grafia de shipping strategy ao longo do guia e incluir esse link também na primeira menção a esse conceito.

> **In VTEX Admin:** Store Settings > Shipping > Settings/Geolocation Shipping/Holidays

### Geolocation shipping (polygons)

[Geolocation shipping](https://help.vtex.com/en/tutorial/gerenciar-geolocalizacao--tutorials_138) is the registration of shipping locations using polygons to delimitate areas, usually for delivering in rural and industrial areas.

* [PUT - Create/update polygon](https://developers.vtex.com/docs/api-reference/logistics-api#put-/api/logistics/pvt/configuration/geoshape)
* [GET - List paged polygons](https://developers.vtex.com/docs/api-reference/logistics-api#get-/api/logistics/pvt/configuration/geoshape)
* [GET - List polygon by ID](https://developers.vtex.com/docs/api-reference/logistics-api#get-/api/logistics/pvt/configuration/geoshape/-polygonName-)
* [DELETE - Delete polygon](https://developers.vtex.com/docs/api-reference/logistics-api#delete-/api/logistics/pvt/configuration/geoshape/-polygonName-)

### Reservations

> [Reservation](https://help.vtex.com/en/tutorial/how-does-reservation-work--tutorials_92) is the solution that prevents the same item from being sold more than once. After the shopper completes the purchase, the item bought status changes from `Available` to `Reserved` in the [inventory management](https://help.vtex.com/en/tutorial/inventory-management--tutorials_139).

Reviewer: ```suggestion
[Reservation](https://help.vtex.com/en/tutorial/how-does-reservation-work--tutorials_92) is the solution that prevents the same item from being sold more than once. After the shopper completes the purchase, the status of the purchased item changes from `Available` to `Reserved` in the [inventory management](https://help.vtex.com/en/tutorial/inventory-management--tutorials_139).
```

* [POST - Add blocked delivery windows](https://developers.vtex.com/docs/api-reference/logistics-api#post-/api/logistics/pvt/configuration/carriers/-carrierId-/adddayofweekblocked)
* [GET - Retrieve blocked delivery windows](https://developers.vtex.com/docs/api-reference/logistics-api#get-/api/logistics/pvt/configuration/carriers/-carrierId-/getdayofweekblocked)
* [POST - Remove blocked delivery windows](https://developers.vtex.com/docs/api-reference/logistics-api#post-/api/logistics/pvt/configuration/carriers/-carrierId-/removedayofweekblocked)
* [GET - Search capacity reservations in time range](https://developers.vtex.com/docs/api-reference/logistics-api#get-/api/logistics-capacity/resources/carrier@-capacityType-@-shippingPolicyId-/time-frames)
* [GET - Get capacity reservation usage by window](https://developers.vtex.com/docs/api-reference/logistics-api#get-/api/logistics-capacity/resources/carrier@-capacityType-@-shippingPolicyId-/time-frames/-windowDay-F-windowStartTime-T-windowEndTime-)

### Holidays

> The [Holidays](https://help.vtex.com/en/tutorial/registering-holidays--2ItOthSEAoyAmcwsuiO6Yk) feature allows you to configure days that should not be considered valid for delivery so that VTEX platform considers those days when calculating the shipping estimated time.

Reviewer: ```suggestion
The [Holidays](https://help.vtex.com/en/tutorial/registering-holidays--2ItOthSEAoyAmcwsuiO6Yk) feature allows you to configure days that should not be considered valid for delivery, so that the VTEX platform adds those days when calculating the shipping estimated time.
```
